keyboards/chromatonemini/chromatonemini.c

```diff
@@ -0,0 +1,191 @@
+  { 8, 14, 20, 45, 51, 57,  82,  88,  94, NO_LED, NO_LED, NO_LED},  //  G
+  {25, 31, 37, 62, 68, 74,  99, 105, 111, NO_LED, NO_LED, NO_LED},  //  G#
+  { 9, 15, 21, 46, 52, 58,  83,  89,  95, NO_LED, NO_LED, NO_LED},  //  A
+  {24, 30, 36, 61, 67, 73,  98, 104, 110, NO_LED, NO_LED, NO_LED},  //  A#
+  {10, 16, 22, 47, 53, 59,  84,  90,  96, NO_LED, NO_LED, NO_LED}   //  B
+};
+
+const uint8_t led_single_col_indicator[37][3] = {
+  {41, 78, 115},  //  K01  = C1
+  { 5, 42,  79},  //  K02  = C#1
+  {40, 77, 114},  //  K03  = D1
+  { 6, 43,  80},  //  K04  = D#1
+  {39, 76, 113},  //  K05  = E1
+  { 7, 44,  81},  //  K06  = F1
+  {38, 75, 112},  //  K07  = F#1
+  { 8, 45,  82},  //  K08  = G1
+  {37, 74, 111},  //  K09  = G#1
+  { 9, 46,  83},  //  K10  = A1
+  {36, 73, 110},  //  K11  = A#1
+  {10, 47,  84},  //  K12  = B1
+  {35, 72, 109},  //  K13  = C2
+  {11, 48,  85},  //  K14  = C#2
+  {34, 71, 108},  //  K15  = D2
+  {12, 49,  86},  //  K16  = D#2
+  {33, 70, 107},  //  K17  = E2
+  {13, 50,  87},  //  K18  = F2
+  {32, 69, 106},  //  K19  = F#2
+  {14, 51,  88},  //  K20  = G2
+  {31, 68, 105},  //  K21  = G#2
+  {15, 52,  89},  //  K22  = A2
+  {30, 67, 104},  //  K23  = A#2
+  {16, 53,  90},  //  K24  = B2
+  {29, 66, 103},  //  K25  = C3
+  {17, 54,  91},  //  K26  = C#3
+  {28, 65, 102},  //  K27  = D3
+  {18, 55,  92},  //  K28  = D#3
+  {27, 64, 101},  //  K29  = E3
+  {19, 56,  93},  //  K30  = F3
+  {26, 63, 100},  //  K31  = F#3
+  {20, 57,  94},  //  K32  = G3
+  {25, 62,  99},  //  K33  = G#3
+  {21, 58,  95},  //  K34  = A3
+  {24, 61,  98},  //  K35  = A#3
+  {22, 59,  96},  //  K36  = B3
+  {23, 60,  97}   //  K37  = C4
+};
+
+#endif  //  RGB_MATRIX_ENABLE
+
+uint8_t shift_led_indicator_left(uint8_t scale_indicator_col){
+    if (scale_indicator_col > 0) {
+        scale_indicator_col--;
+    } else {
+        scale_indicator_col = 11;
+    }
+    return scale_indicator_col;
+}
+
+uint8_t shift_led_indicator_right(uint8_t scale_indicator_col){
+    if (scale_indicator_col < 11) {
+        scale_indicator_col++;
+    } else {
+        scale_indicator_col = 0;
+    }
+    return scale_indicator_col;
+}
+
+
+#ifdef ENCODER_ENABLE
+const uint16_t rt_matrix[2][2] = {
+    {1, 7}, {2, 7}
+};
+
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (index == 0) {
+        keypos_t key;
+        int cw = 0;
+        cw = clockwise ? 1 : 0;
+        key.row = rt_matrix[cw][0];
+        key.col = rt_matrix[cw][1];
+        uint8_t layer = layer_switch_get_layer(key);
+        uint16_t keycode = keymap_key_to_keycode(layer, key);
+        keyrecord_t record;
+        record.event.key = key;
+
+        if (keycode < MI_ON){
+            tap_code16(keycode);
+        } else {
+            record.event.pressed = true;
+            process_midi(keycode, &record);
+            for (uint16_t i = TAP_CODE_DELAY; i > 0; i--) {
+                wait_ms(1);
+            }
```

Comment:
You should be fine just doing this. 
```suggestion
            wait_ms(TAP_CODE_DELAY);
```

Reply:
I just referred to the tap_code16(), but now I found it's fixed as you suggested. This is fixed now.